.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: check-yaml
+      - id: debug-statements
+      - id: end-of-file-fixer
+        types: [python]
+      - id: flake8
+        args: ["--isolated", "--ignore=A001, P101, P103, W503"]
+        additional_dependencies:
+          - flake8-string-format
+          - flake8-comprehensions
+          - flake8-builtins
+      - id: requirements-txt-fixer
+      - id: trailing-whitespace
+        types: [python]
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+        args: ["--line-length", "79"]
```

Comment:
IMO we should stick with black defaults, which allows line lengths upto 88.

Reply:
Ok, but it's one or the other. Because flake8 is checking to have line length equal to 79.  So either I limit Black or I tweak flake8 config otherwise they will fight each other :laughing:

Reply:
Yes, I think 88 is better 79. `black` has a flake8 config [here](https://github.com/psf/black/blob/master/docs/compatible_configs.md#flake8).

Reply:
Sure, it's fixed.